nodejs/webrtc-hello-world/package.json

```diff
@@ -12,13 +12,13 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
+    "@bandwidth/webrtc": "^0.4.1",
     "@types/express": "^4.17.2",
-    "@types/jwt-decode": "^2.2.1",
+    "@types/node": "^14.14.14",
```

Comment:
Had to install this in order to use `require()` in our Typescript server to pull in the browser SDK